Proyecto.py

```diff
@@ -1516,6 +1517,13 @@ def poscision_elementos_preg6_rect_malo2(self, superficie):
             
             
 def main():
+    pygame.init()
```

Comment:
All calls to pygame module initialization are handled by passing the modules to PygameCanvas, so these calls should be avoided

Reply:
Okay thanks